apps/emqx/src/emqx_persistent_session_ds/session_internals.hrl

```diff
@@ -19,6 +19,8 @@
 -include("emqx_persistent_message.hrl").
 -include("emqx_durable_session_metadata.hrl").
 
+-define(PERSISTENT_SESSION_DB, emqx_persistent_session).
```

Comment:
DURABLE_SESSION_META or DURABLE_SESSION_STATE?

Reply:
Renamed to the latter.